### DISCUSSION TITLE
Cannot Backup Container Manager

### DISCUSSION THREAD

**Discussion by stevenhaskell** [007revad/Synology_app_mover#118] [Q&A] [answered]
Running into a bit of an issue when backing up Container Manager. 

I can successfully export all of the container settings but then I get a warning stating that there is not enough space. I checked the quota for the backup target and there's more than enough room. Oddly enough the warning is showing the backup destination as blank even though I set it in the config file. Anyone else running into this issue?

Output from the terminal below:

> Synology_app_mover v4.0.63
> DS423+ DSM 7.2.1-69057-5 
> 
> Running from: /volume1/Software/Synology/Scripts/Synology_app_mover-4.0.63/syno_app_mover.sh
> 
> 1) Move
> 2) Backup
> 3) Restore
> Select the mode: 2
> You selected Backup
> 
> Backup path is: /volume1/Backups/Steven/Synology_Apps
> 
> Do you want to backup All packages? [y/n]
> n
> [Installed package list]
>  1) /volume1  Container Manager
>  2) /volume1  Document Viewer
>  3) /volume1  Hyper Backup
>  4) /volume1  Hyper Backup Vault
>  5) /volume1  Log Center
>  6) /volume1  Storage Analyzer
>  7) /volume1  Synology Application Service
>  8) /volume1  Synology Drive Server
>  9) /volume1  Synology Photos
> 10) /volume1  Text Editor
> 11) /volume1  Universal Viewer
> 12) /volume1  VPN Server
> 13) /volume2  Plex Media Server
> Select the package to backup: 1
> You selected Container Manager in /volume1
> 
> Ready to Backup Container Manager to /volume1/Backups/Steven/Synology_Apps? [y/n]
> y
> 
> Starting Container Manager             
> Waiting for Container Manager to start            
> Exporting container settings to /volume1/docker/app_mover_exports
> Exporting vaultwarden-server json
> Exporting HomeAssistant json
> Exporting gitea json
> Exporting flaresolverr json
> Exporting couchdb json
> Exporting tautulli json
> Exporting sonarr json
> Exporting radarr json
> Exporting Prowlarr json
> Exporting unifi-controller json
> Exporting synology_docviewer_2 json
> Exporting synology_docviewer_1 json
> Exporting penpot-frontend json
> Exporting penpot-backend json
> Exporting penpot-db json
> Exporting penpot-redis json
> Exporting penpot-exporter json
> Exporting haugene-transmission-openvpn json
> Exporting speedtest json
> Exporting Requestrr json
> Exporting wordpress-portfolio-site-phpmyadmin json
> Exporting wordpress-portfolio-site json
> Exporting wordpress-portfolio-site-db json
> Exporting readarr json
> Removing dangling docker images
> Stopping Container Manager             
> Waiting for Container Manager to stop            
> WARNING Not enough space to backup /volume1/@docker to 
> Free: 0 GB  Needed: 134 GB (plus 50 GB buffer)
> 
> ERROR Not enough space on  to backup @docker!

**Comment by 007revad** [comment]
@stevenhaskell 

I'm getting the exact same error and think I have found the problem, which was 4 lines of code I commented out, but can't remember why. I'll upload v4.0.65 shortly.

**Reply by stevenhaskell** [reply]
@007revad thank you for looking into this so quickly!

**Reply by 007revad** [reply, marked as answer]
It would seem quick for you, but I've been working on this issue for 2 days already.

I've fixed it in https://github.com/007revad/Synology_app_mover/releases/tag/v4.0.65

**Reply by stevenhaskell** [reply]
@007revad thank you, I appreciate the fix!
